compiler-rt/test/profile/instrprof-api.c

```diff
@@ -19,22 +19,22 @@ __attribute__((noinline)) int bar() { return 4; }
 
 int foo() {
   __llvm_profile_reset_counters();
-  // PROFGEN: call void @__llvm_profile_reset_counters()
-  // PROFUSE-NOT: call void @__llvm_profile_reset_counters()
+  // PROFGEN: call {{(arm_aapcs_vfpcc )*}}void @__llvm_profile_reset_counters()
```

Comment:
nit: Any reason to use `*` instead of `?`?

Reply:
You’re quite observant! No, there’s no specific reason for it; I did this file first, and it used other similar patterns with `*` (that also probably could be `?`) so I matched that style of the surrounding file - then the other file has preexisting regexes with `?` instead, so I matched that there.

I can change both files to use `?` for the new patterns for consistency.